docs/src/styles/reset.css

```diff
@@ -4,6 +4,12 @@ body {
   padding-top: 49px;
 }
 
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
```

Comment:
Great. In the visual regression, we do the opposite, so a great way to ensure that our style works with both models:

https://github.com/mui/base-ui/blob/d29192a7bfaf32dd932a230647d41ed76d2c9f4f/test/regressions/TestViewer.js#L52